### DISCUSSION TITLE
有官方的微信群或QQ群吗

### DISCUSSION THREAD

**Discussion by xuzhongpeng** [openkraken/kraken#564] [General]
有官方的微信群或QQ群吗

**Comment by wssgcg1213** [comment]
暂时没有建立官方的即时通讯群组, 原因是

1. 我们认为使用 Github 提供的 Issue/PR/Discussion 在当下已经满足正常沟通的需求了，有任何问题/讨论都可以通过上述方式与相关协作者取得联系
2. 群组是相对封闭和私有的空间，使用公开讨论的方式便于已经解决的问题被后来的人检索到
